docfx_project/examples/LogCDPCommunication.md

```diff
@@ -0,0 +1,66 @@
+# How to log messages between Puppeteer-Sharp and Chromium
```

Comment:
We need to update the toc https://github.com/hardkoded/puppeteer-sharp/blob/master/docfx_project/examples/toc.yml#L20